frontend/src/app/main/ui/workspace/tokens/sets.cljs

```diff
@@ -445,23 +445,32 @@
          :on-edit-submit-set on-update-token-set
          :on-edit-submit-group on-update-token-set-group}])]))
 
+(mf/defc sets-list*
```

Comment:
Small question on the component name. 
What does the asterisk at the end of the name signify?

Reply:
It has two meanings:
1. Convention: a clear visual distinction between components and functions.
2. Call convention: makes it mandatory to use `:>` for instantiate that components to react elements and dictates how props are destructured. In this case it means `::mf/props :react`, so props are received as js object with react friendly casing (they can be destructured using clojure style, using kebab-kase, or as-is with camelCase, native to react style props). This allows destructure part of props, and pass the rest or transformed with (mf/spread-props...) to the underlying components. It also facilitates expose components to external systems like storybook and make it usable with JSX (which pases props using react camelCase style)